x-pack/plugins/ml/server/models/data_recognizer/modules/siem_auditbeat/ml/linux_rare_kernel_module_arguments.json

```diff
@@ -0,0 +1,53 @@
+{
+    "job_type": "anomaly_detector",
+    "description": "Security: Auditbeat - Looks for unusual kernel modules which are often used for stealth.",
+    "groups": [
+      "security",
+      "auditbeat",
+      "process"
+    ],
+    "analysis_config": {
+      "bucket_span": "15m",
+      "detectors": [
+        {
+          "detector_description": "rare by \"process.title\"",
+          "function": "rare",
+          "by_field_name": "process.title"
+        }
+      ],
+      "influencers": [
+        "process.name",
+        "process.working_directory",
```

Comment:
It's my understanding that for this analysis, it's valuable to know the location (i.e. `process.working_directory`) from which the process is being executed. @randomuserid, if we had to choose between swapping either `process.name` or `process.working_directory` for `process.title` as an influencer, which do you think should go? Here's some example data:

```
"process" : {
            "title" : "/bin/sh -e /usr/share/initramfs-tools/hooks/kmod prereqs",
            "name" : "kmod",
            "working_directory" : "/"
```

Reply:
Note that if we choose to swap out `process.name` then the two custom URLs that use `process.name` should be deleted (unless they also work with `process.title`?).

Reply:
Process.name is probably the best one to cut because it will tend to show up in the process.title field & the data feed is limited to a few kernel module related processes by name so process name will always be one of those.

Reply:
spoke with @randomuserid. `process.name` is limited to just 4 processes because of the datafeed query, so let's swap it out for `process.title` in the influencer list. As @peteharverson mentioned, this means we'll need to remove the custom urls that utilize `process.name`. I'll make the change.